src/DIRAC/__init__.py

```diff
@@ -88,19 +91,31 @@
 # Importing _strptime before instantiating the threads seem to be a working workaround
 import _strptime
 
-
 # Define Version
+if six.PY3:
+  from pkg_resources import get_distribution, DistributionNotFound
 
-majorVersion = 7
-minorVersion = 2
-patchLevel = 0
-preVersion = 24
-
-version = "v%sr%s" % (majorVersion, minorVersion)
-if patchLevel:
-  version = "%sp%s" % (version, patchLevel)
-if preVersion:
-  version = "%s-pre%s" % (version, preVersion)
+  try:
+    __version__ = get_distribution(__name__).version
+    version = __version__
+  except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
maybe set `version = 'Unreleased'` or something like that ?

Reply:
I'll put `Unknown` as it actually means that it hasn't been installed. (Even a `pip install -e .` installation will be given a meaningful version number by `setuptools_scm`.)

Reply:
ah cool I did not know that !